db/migrate/20230912081147_add_subject_limited_to_countries.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubjectLimitedToCountries < ActiveRecord::Migration[7.0]
+  def change
+    add_column :countries, :subject_limited, :boolean, default: false
```

Comment:
```suggestion
    add_column :countries, :subject_limited, :boolean, default: false, null: false
```

I think we should make sure this field can't be `null`, as currently this field can have three values (`true`, `false` and `null`), but we only need to `true` or `false`.